contrib/charts/dragonfly/templates/_pod.tpl

```diff
@@ -123,6 +130,31 @@ containers:
     envFrom:
       {{- toYaml . | trim | nindent 6 }}
     {{- end }}
+  {{- if .Values.snapshot.enabled }}
+  - name: snapshot-cleaner
+    image: {{ .Values.snapshot.cleaner.image }}
+    command:
+      - /bin/sh
+      - -c
+      - |
+        while true; do
+          {{- if eq .Values.snapshot.format "dragonfly" }}
```

Comment:
should be "df" instead of "dragonfly" as you are keeping only two formats "df" and "rdb" from the comment below.